staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -139,6 +147,12 @@ type LeaderElector struct {
 	// value observedRecord.HolderIdentity if the transition has
 	// not yet been reported.
 	reportedLeader string
+
+	// timer is wrapper around time to allow for less flaky testing
+	timer clock.Clock
```

Comment:
I'd actually call it clock, not timer, since a timer is a thing that a clock can create for you.

Reply:
Had been concerned about conflicts between the imported package name and the variable name.